docs/getting-started/setup/optimizing-build-size.md

```diff
@@ -85,7 +56,7 @@ import '@ckeditor/ckeditor5-table/dist/index.css';
 // ...
 ```
 
-Regarding plugin styles, the rule of thumb is to import the styles from all the individual packages from which you import the editor features. For example, if you import the bold feature from `@ckeditor/ckeditor5-basic-styles/dist/index.js`, you should also import `@ckeditor/ckeditor5-basic-styles/dist/index.css`.
+The rule of thumb is to import styles from all the individual packages from which you import the editor features. In your IDE you likely can <kbd>Ctrl</kbd>+<kbd>Left click</kbd> the imported feature name to see in which pages they live. For example, if you click the `Bold` feature, it will lead you to the `@ckeditor/ckeditor5-basic-styles` package in `node_modules`. This means that should also import the styles from the `@ckeditor/ckeditor5-basic-styles/dist/index.css` file. Make sure to not to miss any of the plugins or import the same styles multiple times.
```

Comment:
In which packages? Cause "pages" are a bit unclear.

Reply:
Yeah, it should be `package`